README.md

```diff
@@ -190,7 +190,221 @@ path : /path/to/executable-file
 --
 ```
 
+### `assert_file_owner`
 
+Fail if the user is not the owner of the given file.
+
+```bash
+@test 'assert_file_owner() {
+  assert_file_owner /path/to/owner
```

Comment:
This command should have two arguments: 
1. a user that is owner of the file
2. a file where the user is owner of